### PR TITLE
Remove old reports code [SCI-5650]

### PR DESCRIPTION
**Pull request by okriuchykhin**
Jira ticket: [SCI-5650](https://biosistemika.atlassian.net/browse/SCI-5650)

### What was done
Remove old reports code

### CODE REVIEW

File: app/views/reports/elements/_my_module_repository_element.html.erb [hunk 1]
```diff
@@ -1,17 +1,8 @@
-<% my_module ||= @my_module %>
-<% element_id ||= repository&.id %>
+<% repository ||= report_element.repository %>
```
Comment (aignatov-bio) [inline app/views/reports/elements/_my_module_repository_element.html.erb:1]: For deleted repositories, `report_element` will be `nil`. We pass only `repository_snapshot`

**Review by aignatov-bio — CHANGES_REQUESTED** [review]
Can you rebase and test how it will work with snapshots from deleted repositories,

File: app/views/reports/elements/_my_module_repository_element.html.erb (hunk 1 above)
Comment (okriuchykhin) [reply app/views/reports/elements/_my_module_repository_element.html.erb:1]: What? Why would you even create report element for deleted inventory?

Comment (aignatov-bio) [reply app/views/reports/elements/_my_module_repository_element.html.erb:1]: I mean if my_module has a snapshot from a deleted repository. Repository itself not exist. But we pass snapshot.

Comment (okriuchykhin) [reply app/views/reports/elements/_my_module_repository_element.html.erb:1]: Report element should have connection to the repository, even if it is a snapshot

Comment (aignatov-bio) [reply app/views/reports/elements/_my_module_repository_element.html.erb:1]: Then we need to introduce a new report element - repoistory_snapshot. And create it for each my_module.

File: app/helpers/reports_helper.rb [hunk 2]
```diff
@@ -3,90 +3,24 @@
...
-        children_html.safe_concat render_new_element(true)
-        element.children.active.each do |child|
-          children_html
-            .safe_concat render_report_element(child, provided_locals)
-        end
-      else
-        children_html.safe_concat render_new_element(false)
+    if element.children.active.present?
+      element.children.active.each do |child|
+        children_html.safe_concat render_report_element(child, provided_locals)
```
Comment (artoscinote) [inline app/helpers/reports_helper.rb:17]: `element.children.active&.each do |child|`
would do the same without the if block

**Review by artoscinote — APPROVED** [review]

**Pull request merged**